libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,7 +1,11 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
     <div>
-      <img class="reading-list-item--cover" [src]="b.coverUrl" />
+      <img
+        class="reading-list-item--cover"
+        [src]="b.coverUrl"
+        alt="Reading list cover URL"
```

Comment:
You are providing alt attribute to define the book not image url correct?
So you can provide it a meaningful name like alt="<Book name> book image" or alt="Book image"